### PR TITLE
add test for Form::saveForm

### PR DESCRIPTION
**Pull request by orthagh**
no tested locally, waiting for travis :crossed_fingers:

### CODE REVIEW

File: tests/src/CommonTestCase.php [hunk 1]
```diff
@@ -158,6 +158,9 @@ protected function getForm($input = []) {
       if (!isset($input['name'])) {
          $input['name'] = $this->getUniqueString();
       }
+      if (!isset($input['is_active'])) {
+         $input['is_active'] = true;
```
Comment (btry) [inline tests/src/CommonTestCase.php]: it should be '1', not true IMO

File: tests/suite-unit/PluginFormcreatorForm.php [hunk 2]
```diff
@@ -445,4 +445,70 @@ public function testgetFormFromSection($section, $expected) {
...
+
+      // prepare input
+      $input = [
+         'formcreator_form' => $form->getID(),
+         'formcreator_field_'.$question->getID() => $answer
+      ];
+
+      // send for answer
+      $formAnswerId = $form->saveForm($input);
+      $this->boolean((bool)$formAnswerId)->isTrue();
```
Comment (btry) [inline tests/suite-unit/PluginFormcreatorForm.php]: if a success is expected it is probably better ti check that $formAnswerId is an integer ?

(the method returns a bool only on failure)

Comment (orthagh) [reply tests/suite-unit/PluginFormcreatorForm.php]: the cast bool do the same thing, but i can if you prefer

Comment (orthagh) [reply tests/suite-unit/PluginFormcreatorForm.php]: this is how we do in GLPI test, we don't care to check the integer, the goal is indeed to test if we don't have a false return

Comment (btry) [reply tests/suite-unit/PluginFormcreatorForm.php]: The following should to the job as well

```suggestion
      $this->integer($formAnswerId);
```

**Pull request merged**